meta-hp/recipes-core/systemd/systemd-machine-units_1.0.bbappend

```diff
@@ -1,11 +1,11 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-SRC_URI:append:tenderloin = " \
+SRC_URI:append:tenderloin-halium = " \
```

Comment:
this one should be left as-is, as you already added the append for tenderloin-halium below